lib/main.dart

```diff
@@ -22,70 +24,83 @@ import 'package:quria/presentation/var/routes.dart';
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
-  StorageService.init();
+  await StorageService.init();
```

Comment:
To increase the app performance it might be better to avoid awaiting init if it's not deemed necessary